ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -692,7 +691,7 @@ public PeerDiscoveryController build() {
     }
 
     private void validate() {
-      validateRequiredDependency(keypair, "KeyPair");
+      validateRequiredDependency(nodeKey, "KeyPair");
```

Comment:
does the description meant to be `NodeKey` instead of `KeyPair`?

Reply:
nodeKey - oops.